docs/modules/ROOT/pages/api/testing.adoc

```diff
@@ -118,19 +118,19 @@ A module containing utilities that simplify declaring and deploying contracts us
 [[testing-deployment-declare_class]]
 ==== `[.contract-item-name]#++declare_class++#++(contract_name: ByteArray) → ContractClass++` [.item-kind]#function#
 
-Declares a contract with a `snforge` `declare` call and unwraps the result.
+Declares a contract with a `snforge` `declare` call and unwraps the result. This function will skip declaration and just return the `ContractClass` if the contract is already declared(the result of `snforge` `declare` call is of type `DeclareResult::AlreadyDeclared`).
```

Comment:
Yeah, this looks a bit cleaner. I updated it in this file and also in the code comments. One small change to be more precise is that I used `snforge_std::declare` instead of `snforge:declare`